vendor/profile.ps1

```diff
@@ -41,6 +41,8 @@ try {
     # Check if git is on PATH, i.e. Git already installed on system
     Get-command -Name "git" -ErrorAction Stop >$null
 } catch {
+    $env:Path += $(";" + $env:CMDER_ROOT + "\vendor\git-for-windows\cmd")
+    # for bash.exe, which in the cmd version is found as <GIT>\usr\bin\bash.exe
     $env:Path += $(";" + $env:CMDER_ROOT + "\vendor\git-for-windows\bin")
```

Comment:
I'm actually not sure if this is a nice thing to keep the `...\bin` version: the problem is that if the user has added git to the path (via the git for windows installer), then bash is not found as the installer does not add bin to the path.